site.yml

```diff
@@ -31,6 +31,11 @@
       become: yes
       tags: puma
 
+    - role: jobs
+      become: yes
+      become_user: "{{ app_user }}"
+      tags: jobs
+
```

Comment:
It's seems like a lot of overhead to create a new role for this. Should we just add the code to the `fairfood` role installing the app?

Reply:
🤷 , I like that it's divided into separate files, each with a clear purpose. But it means more files to wade through.
Now that you've already created it, I'd keep it.